src/Microsoft.ServiceFabric.Actors/Runtime/ActorManager.cs

```diff
@@ -793,14 +793,14 @@ private void ArmGcTimer()
 
                 if (this.isClosed && (this.activeActors.Count == 0))
                 {
-                    // all actors are garbaged collected and we are closed
+                    // All actors are garbaged collected and we are closed
                     // no need to schedule the timer again.
                     this.gcTimer.Dispose();
                     this.gcTimer = null;
                 }
                 else
                 {
-                    // Add some randomness to gcTimer firing to handle scenarios in which a Timer firing at exact same interval as
+                    // Adds some randomness to gcTimer firing to handle scenarios in which a Timer firing at exact same interval as
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.